### DISCUSSION TITLE
Perfera Floor Standing Instructions

### DISCUSSION THREAD

**Discussion by trullock** [revk/xESP32-Faikout#291] [Show and tell]
![PXL_20240403_190430006~2](https://github.com/revk/ESP32-Faikin/assets/118878/bdd21a92-49b6-487e-a582-a496664a6324)
Slide the catches to release the front, unhook the string and lift off

![PXL_20240403_190351979~2](https://github.com/revk/ESP32-Faikin/assets/118878/9c78e793-56a8-4367-a286-18bf7b1afd93)
Undo these four screws and remove the front cover, its lightly clipped on at the top behind the vent, gentle tugging should easily release it

![PXL_20240403_190053156~2](https://github.com/revk/ESP32-Faikin/assets/118878/6d48ac28-5e50-4eb2-ab2f-1db86fef8d9c)
Remove these two screws

![PXL_20240403_190045161](https://github.com/revk/ESP32-Faikin/assets/118878/9e247ff7-9c6f-4083-b033-48f5a5bc70c8)
Lift this up and carefully remove the first metal case by jiggling it to the right, it encloses live mains so be careful if you havent turned the power off

![PXL_20240403_190024066](https://github.com/revk/ESP32-Faikin/assets/118878/b5d94f2a-9181-4433-b9a9-15487a270f77)
Remove the second metal case by jiggling it to the right, again beware exposed mains

![PXL_20240403_185749018](https://github.com/revk/ESP32-Faikin/assets/118878/9d39c593-5c3f-46c0-94d3-19a4c626ec91)
The existing wifi board lives in this little compartment at the top, unclip and unplug it, making space for Faikin

![PXL_20240403_185956992 MP~2](https://github.com/revk/ESP32-Faikin/assets/118878/6080cca9-4454-4d5c-9eee-da23b458e779)
Faikin in its new home, all plugged in.

As you look at the S21 connector face on, in the orientation its mounted, GND is on the right, with 5V on the left.

I used these cables @30cm with female dupont: https://www.aliexpress.com/item/1005006291181398.html
You can probably use 20cm for the Floor Standing Perferas but you need 30cm for the Wall Mounted Perferas

Hope this helps someone

**Comment by ThiefMaster** [comment]
Is there a good space to put the faikin w/o removing the original wifi board?

**Reply by trullock** [reply]
Not in the same compartment, but you could just let it dangle out the back, at approx 2pm on my red "circle" on the last pic, theres room there

**Reply by DragonQ** [reply]
I followed this guide to install the Faikin into my 2023 Perfera and installed it here, seems to work fine.
![PXL_20240507_104241275_small](https://github.com/revk/ESP32-Faikin/assets/3251396/f10f1e7a-1c40-49af-aad1-3ce7d64dcde3)
![PXL_20240507_104250717_small](https://github.com/revk/ESP32-Faikin/assets/3251396/ecc08788-4332-40a2-a929-6b7bb2c8bce5)

**Comment by ThiefMaster** [comment]
> it encloses live mains so be careful if you havent turned the power off

I recommend adding a big fat warning on top to CUT POWER (turning off the breaker, not just turning off the unit)!

**Reply by Y0Sam** [reply]
See my post: https://github.com/revk/ESP32-Faikin/discussions/49#discussioncomment-8192319

**Reply by Y0Sam** [reply]
On the images where the device is placed you see that  I routed the cable to the frond and under need the receiver case. With that you do not need to remove the original wifi module. I do not use it and thus it is not activated.

**Reply by trullock** [reply]
~You have the wall mounted Perfera though, no? This is for the floor standing variety.~

Oh apologies, thats the earlier post

**Reply by Y0Sam** [reply]
It is the version that is called a floor unit but it can also be mounted to a wall if needed/wanted.

**Reply by trullock** [reply]
Well, I've removed it. Don't fancy them forcibly pushing some firmware down that breaks things
